haapi/example-config-template.xml

```diff
@@ -74,8 +74,10 @@
                                 <scope>phone</scope>
                                 <scope>profile</scope>
                                 <capabilities>
-                                <code/>
-                                <haapi/>
+                                    <code/>
+                                    <haapi>
+                                        <issue-token-bound-authorization-code>true</issue-token-bound-authorization-code>
```

Comment:
Should we add a note somewhere that you need to use at least version 8.7 of the Curity Identity Server? The scripts will not work now with older versions, right?

Reply:
Will ensure added to all HAAPI example repos which use this template